src/config/env.js

```diff
@@ -11,19 +11,21 @@
     "brokerUrl": "http://localhost:8080/broker",
     "sampleLocatorUrl": "http://localhost:8085",
     "negotiatorUrl": "https://negotiator.bbmri-eric.eu",
-    "terminologyServerUrl": "https://r4.ontoserver.csiro.au/fhir"
+    "terminologyServerUrl": "https://samplelocator.bbmri.de/icd10"
```

Comment:
This change is already present in the master. Please merge the master in develop and rebase onto it.